packages/node-resizin/README.md

```diff
@@ -51,7 +51,7 @@ upload(file).then(() => {
 *  There is also third parametr, available only for Node.js, that determines file mime type. The mime type is optional and defaults to `application/octet-stream`.
 
     ```js
-    const { uploadFactory } = require('node-resizin');
+    const uploadFactory = require('node-resizin');
```

Comment:
this import is changed from named import to default import - but I cant see any change in the code or update version in package.json

It was an old mistake from before just here in the documentation or what?